src/lib/select/select.ts

```diff
@@ -202,6 +202,7 @@ export class MatSelectTrigger {}
     '[class.mat-select-disabled]': 'disabled',
     '[class.mat-select-invalid]': 'errorState',
     '[class.mat-select-required]': 'required',
+    '[class.mat-select-empty]': 'empty',
```

Comment:
iirc, form-field has an empty class that you can probably use instead of adding a new one here

Reply:
Hey, just checked this out, but the .mat-form-field-empty is only set on the [floating label](https://github.com/seritools/material2/blob/63623022ff9665dc4b566b60cdff1fc50fd8f960/src/lib/form-field/form-field.html#L34), not the mat-form-field itself. What would be the best change @mmalerba?

Reply:
Oh ok, in that case we can just leave it as is